### PR TITLE
Updated information on using Xdebug

### PR DESCRIPTION
**Pull request by rimi-itk**
Updates information on using Xdebug in PHPStorm. Adds information on how to use Xdebug in VS Code.

### CODE REVIEW

File: scripts/itkdev-docker-compose [hunk 1]
```diff
@@ -551,27 +551,52 @@ EOF
 
 ================================================================================
 
-To debug from the command line, add `PHP_IDE_CONFIG=serverName=localhost` to the
-`phpfpm` environment:
+Make dure that `PHP_IDE_CONFIG=serverName=localhost` is set in the `phpfpm`
```
Comment (turegjorup) [inline scripts/itkdev-docker-compose:554]: dure -> sure

Comment (rimi-itk) [reply scripts/itkdev-docker-compose]: Nice catch!

**Review by turegjorup — APPROVED** [review]

**Comment by turegjorup** [comment]
Tested instructions on PHPstorm. Can't test/vouch for VScode instructions.

**Comment by rimi-itk** [comment]
> Tested instructions on PHPstorm. Can't test/vouch for VScode instructions.

Both @jeppekroghitk and I have tested them successfully.

**Pull request merged**